### DISCUSSION TITLE
how to enable Amazon Polly Japanese voice?

### DISCUSSION THREAD

**Discussion by meow-master** [karashiiro/TextToTalk#67] [General]
Hello,

I am playing FF14 in Japanese and would love to use Amazon polly's voices as they sound way more natural than Microsoft's Haruka voice that I'm currently using. however, after linking my Amazon Polly to the plugin I couldn't figure out how to use the Japanese voices. It seems it is only automatically unable the English voice that read Japanese text... weirdly. 

can you help me to enable Amazon Polly's Japanese voices? I will be very very appreciated! thank you very much.

**Comment by beckyeeky** [comment]
Try switch the Region in settings to ap-northeast-1, which means tokyo center.
the list of voice available varies from region to region.

~~I myself have another issue that I cannot find voice for female (Mizuki) in the list ...~~
Also switch Engine setting to "standard" so non-neural voice will also show up.

**Reply by meow-master** [reply]
oh my god, it worked and it sounds so much better!  thank you so much.

**Comment by karashiiro** [comment]
It's also possible that you have a voice engine enabled that doesn't support it. Try switching to the Neural engine if you're not using it.

**Reply by meow-master** [reply]
the Neural engine definitely adds the naturity into the voices which I really love! sadly it doesn't support the female Japanese sound, only one gender, male.  it is still way better than what I used to use! thank you so much for your help :>
